### PR TITLE
Use the system font for header elements in popups

### PR DESCRIPTION
**Pull request by rwols**

### CODE REVIEW

File: popups.css [hunk 1]
```diff
@@ -13,6 +13,12 @@
 .lsp_popup li {
     font-family: system;
 }
+.lsp_popup h1,
+.lsp_popup h2,
+.lsp_popup h3,
+.lsp_popup h4 {
```
Comment (rchl) [inline popups.css:19]: You could combine those rules to avoid duplication.

**Review by rchl — COMMENTED** [review]
What kind of content uses header elements and should we really apply those to all headers, even ones that come from a server? Or should we even just apply that font style to the whole popup? At this point, I'm not sure what doesn't use it since the `p` element does which is probably used in a lot of places.

File: popups.css (hunk 1 above)
Comment (rwols) [reply popups.css]: I'm not very good at CSS. I think I combined the selectors in my latest commit?

Comment (rchl) [reply popups.css]: Perfect.

**Comment by rwols** [comment]
> What kind of content uses header elements and should we really apply those to all headers, even ones that come from a server?

\<h1\> is used a lot by clangd.

> Or should we even just apply that font style to the whole popup?

I would say *yes*, but I don't know how to do that in a neat way, because I suck at css.

**Comment by rchl** [comment]
> I would say _yes_, but I don't know how to do that in a neat way, because I suck at css.

So we could just have a `font-family: system;` in the top `.lsp_popup` selector. That will be inherited by all children unless something overrides `font-family` (or `font`, if ST supports that) to something else. Also, ST's internal styles might override it if those specify `font-family` for certain elements (but I don't know if they do).

But we can just have `font-family` on the top selector and see if it doesn't apply to something and then make a specific override for that.

**Comment by rwols** [comment]
Hmm... this also styles the diagnostics with the system font. I don't think that's an improvement.

**Review by rchl — APPROVED** [review]
It feels a bit random to me in which font-family style we style what. We target certain element types rather than certain content types.

But I guess it should be fine, we can think of a more generic solution later.

**Comment by rwols** [comment]
I managed to simplify it I think :) or can something go wrong with this?

File: popups.css [hunk 2]
```diff
@@ -1,52 +1,47 @@
...
 }
-.lsp_popup p {
+.diagnostics {
     margin-bottom: 0.5rem;
-    padding: 0 0.5rem;
-    font-family: system;
-}
-.lsp_popup li {
-    font-family: system;
+    font-family: monospace;
```
Comment (rwols) [inline popups.css:11]: I'm not sure how to "reset" the font-family here. The value `monospace` doesn't actually do what it is supposed to do: https://github.com/sublimelsp/LSP/issues/1171#issuecomment-651800108

Comment (rchl) [reply popups.css:11]: You are referring to a deleted line?
Not sure what do you exactly mean since the code doesn't look like that anymore.

Comment (rwols) [reply popups.css:11]: I wrote `font-family: monospace;` here, but that doesn't actually work as you'd expect. It just so happens to work. But the behavior is unconfirmed on Windows.

The only special font-family is presumably `system`.

**Review by rchl — APPROVED** [review]

**Pull request merged**